.github/CODEOWNERS

```diff
@@ -864,6 +864,8 @@
 
 # PRLabel: %Monitor
 /sdk/monitor/ @hectorhdzg @JacksonWeber
+/sdk/monitor/monitor-opentelemetry @hectorhdzg @JacksonWeber
```

Comment:
There's no benefit to these, as the `/sdk/monitor/` path has the same handles.   I'd suggest removing them.

Reply:
@jsquire I though the same, but the reason I added this was to discuss precisely that removing us from the generic path, neither Jackson or I should be reviewing all other Monitor projects in this folder, we do not own or know how they work.

Reply:
so we should remove `/sdk/monitor/` line? we could add more specific entries for sub folders if needed.

Reply:
That would be nice, but I'm not sure who to add for arm-monitor, arm-monitor-profile and monitor-ingestion projects

Reply:
arm-monitor and arm-monitor-profile is already covered as Mgmt packages. monitor-ingestion maybe @KarishmaGhiya?

Reply:
yes please add @KarishmaGhiya @sarangan12 for the other two non-arm sub folders.